crates/ruff_linter/resources/test/fixtures/ruff/RUF029.py

```diff
@@ -0,0 +1,30 @@
+import time
+import asyncio
+
+async def pass_case(): # passes because it awaits something
```

Comment:
Usually we'd indicate this with something like 

```suggestion
async def pass_case(): # OK: awaits a coroutine
```

Reply:
It's not really standardized though, so it's okay either way.

Reply:
I've made this change below.